chat-plugins/info.js

```diff
@@ -388,19 +396,31 @@ exports.commands = {
 					}
 					return this.sendReply(buffer);
 				} else {
-					buffer += '|c|~|/data-' + newTargets[i].searchType + ' ' + newTargets[i].name + '\n';
+					let sType = newTargets[i].searchType.charAt(0).toUpperCase() + newTargets[i].searchType.substring(1, newTargets[i].searchType.length);
+					if (Chat[`getData${sType}HTML`]) {
+						let template = mod["get" + (sType === "Pokemon" ? "Template" : sType)](newTargets[i].name);
+						if (sType === "Pokemon") {
+							if (mod.gen === 1) {
+								template.baseStats["spc"] = template.baseStats.spa;
+							}
+							if (mod.gen < 3) delete template.abilities;
```

Comment:
Ugh, please stop mutating the outputs of dex data functions. This could easily cause other code to crash.